CMakeLists.txt

```diff
@@ -443,6 +431,39 @@ else()
 endif()
 list(APPEND SRC_FILES ${CU_SOURCES})
 
+if (ENABLE_MSCCL)
+  add_definitions(-DCOMPILE_MSCCL)
```

Comment:
This line shouldn't be here.  It already is handled by target_compile_definitions(rccl PRIVATE COMPILE_MSCCL)